tools/sort-fails.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash +x
+
+# Copyright 2015 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+INPUT_PATH="$1"
+VERSION=$(git log -n 1 --pretty=format:"%H")
+SYSTEM=$(uname -a)
+TMP_DIR="afl-testing"
+USAGE="Usage:\n\t./tools/sort-fails.sh PATH_TO_DIR_WITH_FAILS"
+
+if [ "$#" -ne 1 ]
+then
+  echo -e "${USAGE}";
+  echo "Argument number mismatch...";
+	exit 1;
+fi
+
+rm $TMP_DIR -r; mkdir $TMP_DIR;
+
+make debug.linux VALGRIND=ON -j && echo "Build OK";
+
+for file in $INPUT_PATH/*;
+do
+	key1=$(./build/bin/debug.linux/jerry --abort-on-fail $file 2>&1);
+	key2=$(valgrind --default-suppressions=yes ./build/bin/debug.linux/jerry --abort-on-fail $file 2>&1 \
+				| sed -e 's/^==*[0-9][0-9]*==*//g' \
+				| sed -e 's/^ Command: .*//g');
+
+	hash1=$(echo -n $key1 | md5sum);
+	hash2=$(echo -n $key2 | md5sum);
```

Comment:
Quotes are necessary in the case - otherwise newlines are not outputted and so are not considered during hash calculation.